conda/base/constants.py

```diff
@@ -324,12 +324,3 @@ class NoticeLevel(ValueEnum):
 # Indicates whether or not external plugins (i.e., plugins that aren't shipped
 # with conda) are enabled
 NO_PLUGINS = False
-
-#: Name we use to identify the auth backend in ``channel_settings``
```

Comment:
This constants aren't useful here and seem random, and increase the API surface without good cause.